src/flatbuffers/metric.fbs

```diff
@@ -0,0 +1,15 @@
+include "metric_common.fbs";
+
+// Metric message
+namespace circonus;
+
+table Metric {
+      timestamp: ulong (id: 0);
+      check_name: string (id: 1);
+      check_uuid: string (id: 2);
+      account_id: int (id: 3);
+      value:MetricValue (id: 4);
```

Comment:
I'd like to handle that when the tag stuff is closer to completion.. it's a simple addition at that point.